staging/src/k8s.io/apiserver/pkg/util/feature/feature_gate.go

```diff
@@ -70,10 +70,16 @@ const (
 // FeatureGate parses and stores flag gates for known features from
 // a string like feature1=true,feature2=false,...
 type FeatureGate interface {
+	// AddFlag adds a flag for setting global feature gates to the specified FlagSet.
 	AddFlag(fs *pflag.FlagSet)
+	// Set Parses a string of the form // "key1=value1,key2=value2,..." into a
+	// map[string]bool of known keys or returns an error.
 	Set(value string) error
+	// Enabled returns true if the key is enabled.
 	Enabled(key Feature) bool
+	// Add adds a Feature.
```

Comment:
Does this just add one Feature, or multiple? Since `features` is a map, it sounds like multiple.